roles/dlrn_report/tasks/main.yml

```diff
@@ -14,11 +14,21 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Warning if DLRN is not running
+  debug:
+  msg: "DLRN will not be executed if it is a testproject, make sure you set cifmw_dlrn_report_force_report to true if you want to report to DLRN"
```

Comment:
syntax error?

The msg should be nested under `debug`.
Also please use `ansible.builtin.debug` instead of just `debug`.